Watchman.Discord/Integration/DevscordFramework/CustomCommandsLoader.cs

```diff
@@ -42,6 +42,8 @@ public async Task InitDefaultCustomCommands()
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.AddRoleCommand", @"-add\s*role\s*(?<GetRoles>[\w\W\s\""]*)", serverId),
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.RemoveRoleCommand", @"-remove\s*role\s*(?<GetRoles>[\w\W\s\""]*)", serverId),
                     new AddCustomCommandsCommand("Watchman.Discord.Areas.Administration.BotCommands.SetRoleCommand", @"-set\s*role\s*(?<GetRoles>[\w\W\s\""]*)\s*-((?<Safe>safe)|(?<Unsafe>unsafe))", serverId),
+                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.AddWarnCommand", @"-add\s*warn\s*-user\s*(?<User><@\d+>)\s*-reason\s*(?<Reason>.+)", serverId),
```

Comment:
```suggestion
                    new AddCustomCommandsCommand("Watchman.Discord.Areas.Users.BotCommands.Warns.AddWarnCommand", @"-addwarn\s*(?<User><@\d+>)\s*-(reason|r)\s(?<Reason>.+)", serverId),
```

Reply:
\s lepiej zawsze dawać z * 
zawsze to dodatkowa idiotoodporność gdyby ktoś dał podwójną spacje

Reply:
to może lepiej `+`
bo gwiazdka oznacza, że reason może być pusty jeśli wpisze się samą spację po reason

Reply:
dałem z +